test/libsolidity/SemanticTest.cpp

```diff
@@ -581,7 +674,10 @@ void SemanticTest::printUpdatedSettings(ostream& _stream, string const& _linePre
 
 	_stream << _linePrefix << "// ====" << endl;
 	for (auto const& [settingName, settingValue]: settings)
-		_stream << _linePrefix << "// " << settingName << ": " << settingValue<< endl;
+		if (settingName == "requiresYulOptimizer")
+			_stream << _linePrefix << "// " << settingName << ": " << m_requiresYulOptimizer << endl;
```

Comment:
Will this add the ``requiresYulOptimizer`` setting in case it previously wasn't there, but the test fails without it?
If not, if this ever happens, it'd send `--accept-updates` in a loop, won't it?

Reply:
You're right, it won't. But it's fixed now.

I noticed it only after pushing the PR. I decided to retest it manually just in case, since the update is not part of the normal flow when tests pass, and only then found out it was broken :)